pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -61,7 +61,7 @@ func Execute(hnd resourcemonitor.Handle, nrtupdaterArgs nrtupdater.Args, resourc
 	}
 	go resObs.Run(eventSource.Events(), condChan)
 
-	upd := nrtupdater.NewNRTUpdater(nrtupdaterArgs, tmConf.config)
+	upd := nrtupdater.NewNRTUpdater(hnd.K8SCli, nrtupdaterArgs, tmConf.config)
```

Comment:
this should be done conditionally controlled by an option. I'm very wary of adding load to the apiserver

Reply:
Ok, updated in the new commit also.